### DISCUSSION TITLE
Welcome to MIO-KITCHEN-SOURCE Discussions!

### DISCUSSION THREAD

**Discussion by ColdWindScholar** [ColdWindScholar/MIO-KITCHEN-SOURCE#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
we did ***NOT*** intend to include any malicious binaries to our project, if you think we accidentally did, please give us more information so we can investigate @RedKingIncDeveloper

we also test and depend on these helper programs locally, so it will be great if you can spot out us the issue

**Reply by ghost** [reply]
we should probably have these binaries downloaded dynamicatlly, shipping them with the source code was indeed hacky, but that is what we currently have, @ColdWindScholar is lazy to refactor or clean up

**Reply by ghost** [reply]
our project is here for nearly 2 years, plus the release zip is fully reproducible since it was generated by github actions (not on our machines)

**Reply by ghost** [reply]
if you still have security concerns, you can also choose to build the tool from source yourself, which is documented on our README, so you are free to remove anything that you think is "virus"

**Reply by ghost** [reply]
what i recommend you to not do is spreading out rumors without reasonable details which only make the devs feel sad when they put countless hours into making / improving open source projects like this in their own *free time*

**Comment by ColdWindScholar** [comment]
I'm sorry that I banned you and remove your report without any reason. 
You can rejoin because you are not listed in my ban list
I may have been too impulsive, I'm very sorry @RedKingIncDeveloper
